monitoring/grafana/dashboards/examples/queues-example.json

```diff
@@ -856,13 +856,13 @@
             "uid": "fDMLtag4k"
           },
           "editorMode": "builder",
-          "expr": "queue_length{job=\"ar_io_nodes\", queue_name=\"bundleDataImporter\"}",
+          "expr": "queue_length{job=\"ar_io_nodes\", queue_name=\"dataImporter\"}",
           "legendFormat": "{{instance}}",
           "range": true,
           "refId": "A"
         }
       ],
-      "title": "bundleDataImporter",
+      "title": "dataImporter",
```

Comment:
This data import will still be for bundles so the name shouldn't change.

Reply:
done (the dataImporter as prometheus metric exists separately but this file is now unchanged)